styletesla.css

```diff
@@ -1,97 +1,742 @@
+    width: 35px;
+    margin: 3px auto;
+    display: block;
+    height: 3px;
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+    background: #E51937;
+  }
+
+  .header {
+    height: 83px;
+    margin-bottom: 0;
+  }
+
+  .main {
+    background: url('img/tesla-wordmark.png') no-repeat;
+    background-position-y: 6vh;
+    background-size: contain;
+  }
+
+  .main__subtitle {
+    margin-bottom: 12px;
+  }
+
+  .tesla { 
+    width: 95vw;
+  }
+
+  .description {
+    width: 62vw;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-bottom: 25px;
+  }
+
+  .speed {
+    width: 240px;
+  }
+  .road {
+    padding-right: 0;
+  }
+  .wheel__desc {
+    font: 16px/20px montserratbold;
+    color: #E51937;
+  }
+
+  .icon_wheel {
+    display: none;
+  }
+  .icon_redwheel {
+    display: block;
+    width: 30px;
+    height: 30px;
+    align-self: center;
+    padding-right: 15px;
+  }
+
+  .btn__order {
+    width: 220px;
+    height: 55px;
+    background: #E51937;
+    color: white;
+    font: 16px/20px montserratbold;
+  }
+}
+
+@media (max-width: 900px) {
+  .description {
+    width: 83vw;
+  }
+}
+
+@media (max-width: 670px) {
+  .speed {
+    width: 175px;
+    margin-right: 42px;
+  }
+}
+
+@media (max-width: 530px) {
+  .mainpage {
+    height: 600px;
+  }
+  .header {
+    height: 75px;
+  }
+
+  .logo {
+    width: 138px;
+  }
+
+  .menumobile span {
+    width: 24px;
+  }
+
+  .main__title {
+    font: 50px/61px montserratbold;
+  }
+
+  .main__subtitle {
+    font: 10px/12px montserratmedium;
+  }
+
+  .main {
+    background: url('img/tesla-wordmark.png') no-repeat;
+    background-position-y: 20%;
+    background-size: contain;
+  }
+
+  .description {
+    width: 100vw;
+    margin-bottom: 35px;
+  }
+
+  .speed {
+    width: 160px;
+    margin-right: 20px;
+  }
+
+  .road {
+    margin-right: 50px;
+  }
+
+  .icon_speed {
+    width: 38px;
+    height: 28px;
+  }
+
+  .icon_road {
+    width: 22px;
+    height: 35px;
+  }
+
+  .number {
+    font: 27px/20px montserratsemibold;
+  }
+
+  .dimension /*.secunda, .km*/ {
+    font: 15px/14px montserratbold;
+  }
+
+  .text_bottom {
+    font: 10px/10px montserratregular;
+   }
+
+  .icon_redwheel {
+    width: 22px;
+    height: 22px;
+   }
+
+  .wheel__desc {
+    font: 13px/16px montserratbold;
+  }
+
+  .btn__order {
+    width: 160px;
+    height: 40px;
+    font: 13px/16px montserratbold;
+  }
 }
-.first__btn {
-	width: 160px;
-	height: 40px;
-	border: 2px solid red;
-	color: red;
-	text-align: center;
+
+@media (max-width: 370px) {
+  .speed {
+    width: 122px;
+  }
+
+  .wheel {
+    margin-right: 25px;
+  }
+
+  .speed {
+    width: 122px;
+  }
+  
+  .btn__order {
+    width: 140px;
+  }
 }
+
+/*вторая секция*/
 .advantage {
-	height: 100vh;
-	background: url('img/rulj.png') bottom left no-repeat; 
-	display: flex;
-	justify-content: space-between;
+  width: 100%;
+  background: url('img/rulj.png') left bottom no-repeat; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)